src/main/java/emissary/command/ServerCommand.java

```diff
@@ -62,6 +66,11 @@ public boolean shouldDumpJettyBeans() {
         return dumpJettyBeans;
     }
 
+    public boolean shouldStrictMode() {
```

Comment:
I'd include javadoc on this public method about what it means.  It's not obvious to me what "strict mode" would mean without more context.